src/app/components/inline-form/index.html

```diff
@@ -0,0 +1,89 @@
+<sky-demo-page
+  pageTitle="Inline form"
+  skyuxModule=" "
+>
+
+<sky-demo-page-summary>
+	<p>The inline form component renders a form in the current view rather than opening a modal.</p>
+</sky-demo-page-summary>
+	
+<sky-demo-page-content sectionHeading="UX guidelines">
+
+<header><div class="sky-section-heading">Usage</div></header>
+	<header><div class="sky-subsection-heading sky-margin-stacked-default">When to use inline forms</div></header>
+	<stache-row>
+    <stache-column screenSmall="6">
+			<ul>
+			<li>The add/edit form is simple in terms of a small # of fields and low complexity of input controls</li>
+			<li>The surrounding content will not distract the user from the add/edit task</li>
+			<li>There is a high likelihood the user would need to add multiple records in succession</li>
+			<li>Adding/editing a subsection within a larger modal form</li>
+			<li>It’s ok if the user abandons the task while the add/edit form is open (i.e. the task is functionally non-modal)</li>
+			</ul>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-success sky-margin-stacked-default">
+				<img src="~/assets/img/guidelines/inline-form/usage-do-repeater.png" style="width: 100%;" />
+			</div>
+			<div class="screenshot screenshot-success">
+				<img src="~/assets/img/guidelines/inline-form/usage-do-block.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+	</stache-row>
+		
+	
+	
+	<header><div class="sky-subsection-heading sky-margin-stacked-default">When not to use inline forms</div></header>
+	<stache-row>
+    <stache-column screenSmall="6">
+			<ul>
+			<li>Add/edit form would lead to a large amount of page morphing or a user needing to scroll to complete the form if done inline</li>
+			<li>Working with a paginated or infinite scroll list of items</li>
+			<li>You want the user to have to finish the add/edit before doing anything else (i.e. the task is functionally modal)</li>
```

Comment:
I'd change "You want the user to have to" to "Users must" (unless we don't want to state this rule too strongly). ... I'd change "the add/edit" to "the add or edit form." ... I'm also not a huge fan of the parenthetical at the end. Can we change it to "...anything else because the task is functionally modal."?

Reply:
Can probably remove the parenthetical part as in earlier comment. other changes seem good.